File: devices-api-java-mvc/src/main/java/com/lucas/devicesapijavamvc/configuration/OpenApiConfig.java
package com.lucas.devicesapijavamvc.configuration;

import io.swagger.v3.oas.annotations.OpenAPIDefinition;
import io.swagger.v3.oas.annotations.info.Info;
import io.swagger.v3.oas.annotations.servers.Server;
import org.springframework.context.annotation.Configuration;

@Configuration
@OpenAPIDefinition(
        info = @Info(
                title = "Devices API – Java MVC",
                version = "1.0",
                description = "REST API for managing electronic devices using Spring MVC."
        ),
        servers = {
                @Server(
                        url = "http://localhost:8080",
                        description = "Local Java environment"
                )
        }
)
public class OpenApiConfig {
}

File: devices-api-kotlin-reactive/src/main/kotlin/com/lucas/devicesapikotlinreactive/configuration/OpenApiConfig.kt
package com.lucas.devicesapikotlinreactive.configuration

import io.swagger.v3.oas.annotations.OpenAPIDefinition
import io.swagger.v3.oas.annotations.info.Info
import io.swagger.v3.oas.annotations.servers.Server
import org.springframework.context.annotation.Configuration

@Configuration
@OpenAPIDefinition(
    info = Info(
        title = "Devices API – Kotlin Reactive",
        version = "1.0",
        description = "Reactive REST API for managing devices using Spring WebFlux."
    ),
    servers = [
        Server(
            url = "http://localhost:8081",
            description = "Local Kotlin environment"
        )
    ]
)
class OpenApiConfig
